tests/dummy/app/templates/docs/components/amount-input.md

```diff
@@ -0,0 +1,38 @@
+<h1>AMOUNT-INPUT</h1>
+
+<h2>Basic</h2>
+
+<DocsDemo as |demo|>
+  <demo.example @name="amount-input-base">
+    {{amount-input
```

Comment:
I thought this was to be updated to octane format just like the other MRs I reviewed https://github.com/qonto/ember-amount-input/pull/368/files#diff-c95cb8ca3deac9158df4809fce2d1800e5a72097802d10030a72e4af6b305330

Reply:
This MR is only to upgrade `ember-cli-addon-docs` and any changes needed for it to work. I couldn't get the `ember-cli-addon-docs` components in this template to work properly with the curlies syntax, so am using the angle bracket for `ember-cli-addon-docs` components only when needed. The other MR https://github.com/qonto/ember-amount-input/pull/368 is updating syntax for `amount-input`. 

Ideally this change wouldn't be needed in this MR but it was the minimal change in order to update `ember-cli-addon-docs` I could find.

Reply:
Ok that's clear. Asides that its LGTM